src/logging/logger.ts

```diff
@@ -172,11 +173,19 @@ export class Logger implements ZLUX.Logger {
     Logger.offsetMs = offsetMs;
     this.destinations = new Array<(componentName:string, minimumLevel: LogLevel, ...loggableItems:any[])=>void>();    
     this.previousPatterns = new Array<RegExpLevel>();
-    if (!Logger.processString) {
+    this.updateProcessString();
+  }
+
+  updateProcessString(){
```

Comment:
The implementation looks good but since this is typescript and we create a d.ts file, this function looks like it should be marked `private`.